content/en/docs/ops/configuration/traffic-management/multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Multi-cluster Traffic Management
+description: How to configure how traffic is distributed among clusters in the mesh.
+weight: 70
+keywords: [traffic-management,multicluster]
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
+applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
+`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
+load-balanced across all clusters in the mesh for a given service. In some cases, that behavior is not desirable, and
+this document describes the different options for overriding that default.
+
+## Cluster Local Services
```

Comment:
should we also have a section that highlights locality load balancing as an option? and is this header correct?